### PR TITLE
SNMP construct.xml Slightly complete the description

### PR DESCRIPTION
**Pull request by mmalferov**
Maybe we should add some known information to the description of the method?

### CODE REVIEW

File: reference/snmp/snmp/construct.xml [hunk 1]
```diff
@@ -43,7 +43,8 @@
       The <acronym>SNMP</acronym> agent. <parameter>hostname</parameter> may be suffixed with
       optional <acronym>SNMP</acronym> agent port after colon. IPv6 addresses must be enclosed in square
       brackets if used with port. If FQDN is used for <parameter>hostname</parameter>
-      it will be resolved by php-snmp library, not by Net-SNMP engine. Usage
+      it will be resolved by <literal>php-snmp</literal> library, not by
+      <productname>Net-SNMP</productname> engine. Usage
```
Comment (cmb69) [inline reference/snmp/snmp/construct.xml:47]: I wouldn't call it "php-snmp library", but rather "snmp extension", but I don't think this info is relevant for users of that extension. Maybe just `<parameter>hostname</parameter> also supports <acronym>FQDN</acronym>s`.

File: reference/snmp/snmp/construct.xml [hunk 2]
```diff
@@ -43,7 +43,8 @@
...
       of IPv6 addresses when specifying FQDN may be forced by enclosing FQDN
       into square brackets. Here it is some examples:
```
Comment (cmb69) [inline reference/snmp/snmp/construct.xml:49]:
```suggestion
      into square brackets. Here are some examples:
```

File: reference/snmp/snmp/construct.xml [hunk 3]
```diff
@@ -66,17 +67,51 @@
...
+          <literal>authNoPriv</literal> (an auth passphrase and auth protocol are required), or
+          <literal>authPriv</literal> (both an auth passphrase and a protocol, as well as
+          a privacy passphrase and protocol are required)
+         </entry>
+        </row>
        </tbody>
       </tgroup>
      </informaltable>
+     <simpara>
+      SNMPv3 require to configure security-related session parameters with
```
Comment (cmb69) [inline reference/snmp/snmp/construct.xml:112]:
```suggestion
      SNMPv3 requires to configure security-related session parameters with
```

File: reference/snmp/snmp/construct.xml [hunk 4]
```diff
@@ -66,17 +67,51 @@
...
+     </para>
      <informaltable>
       <tgroup cols="2">
        <tbody>
-        <row><entry>SNMP::VERSION_1</entry><entry><acronym>SNMP</acronym> community</entry></row>
-        <row><entry>SNMP::VERSION_2C</entry><entry><acronym>SNMP</acronym> community</entry></row>
-        <row><entry>SNMP::VERSION_3</entry><entry><acronym>SNMP</acronym>v3 securityName</entry></row>
+        <row>
+         <entry>
+          SNMP::VERSION_1
```
Comment (cmb69) [inline reference/snmp/snmp/construct.xml:80]:
```suggestion
          <constant>SNMP::VERSION_1</constant>
```

File: reference/snmp/snmp/construct.xml [hunk 5]
```diff
@@ -66,17 +67,51 @@
...
+         </entry>
+         <entry>
+          <literal>public</literal> for read-only permission or
+          <literal>private</literal> for read-write
+         </entry>
+        </row>
+        <row>
+         <entry>
+          SNMP::VERSION_2C
```
Comment (cmb69) [inline reference/snmp/snmp/construct.xml:89]:
```suggestion
          <constant>SNMP::VERSION_2C</constant>
```

File: reference/snmp/snmp/construct.xml [hunk 6]
```diff
@@ -66,17 +67,51 @@
...
+         </entry>
+         <entry>
+          <literal>public</literal> for read-only permission or
+          <literal>private</literal> for read-write
+         </entry>
+        </row>
+        <row>
+         <entry>
+          SNMP::VERSION_3
```
Comment (cmb69) [inline reference/snmp/snmp/construct.xml:98]:
```suggestion
          <constant>SNMP::VERSION_3</constant>
```

**Review by cmb69 — COMMENTED** [review]
Thank you for the PR! While I think a lot more could be improved on that page (e.g. use `varlist` instead of `informaltable`), that can be done later. A couple of nits, though.

File: reference/snmp/snmp/construct.xml [hunk 7]
```diff
@@ -66,17 +67,51 @@
    <varlistentry>
     <term><parameter>community</parameter></term>
     <listitem>
-     <para>The purpuse of <parameter>community</parameter> is
-     <acronym>SNMP</acronym> version specific:</para>
+     <para>
+      Specifies the security level for given <parameter>version</parameter>.
+      The purpuse of <parameter>community</parameter> access string is
```
Comment (Girgias) [inline reference/snmp/snmp/construct.xml:72]:
```suggestion
      The purpose of <parameter>community</parameter> access string is
```
Typo fix

**Review by cmb69 — APPROVED** [review]
Thank you!

**Pull request merged**